web/src/components/TraceEmptyState.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  Title,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+} from '@patternfly/react-core';
+import CubesIcon from '@patternfly/react-icons/dist/esm/icons/cubes-icon';
+
+import { useTranslation } from 'react-i18next';
+
+export const TraceEmptyState: React.FunctionComponent = () => {
+
+  const { t } = useTranslation('plugin__distributed-tracing-console-plugin');
```

Comment:
My eslint thinks starting the function call with an empty line is an error

```suggestion
export const TraceEmptyState: React.FunctionComponent = () => {
  const { t } = useTranslation('plugin__distributed-tracing-console-plugin');
```